todoman/cli.py

```diff
@@ -53,14 +53,24 @@ def _validate_list_param(ctx, param=None, name=None):
             raise click.BadParameter(
                 'You must set "default_list" or use -l.'.format(name)
             )
-    for l in ctx.db.lists():
-        if l.name == name:
-            return l
-    else:
-        list_names = [l.name for l in ctx.db.lists()]
-        raise click.BadParameter(
-            "{}. Available lists are: {}".format(name, ', '.join(list_names))
-        )
+    lists = {
+        l.name: l
+        for l in ctx.db.lists()
+    }
+    normalized_lists = {
+        l.name.lower(): l
+        for l in lists.values()
+    }
+    if len(lists) == len(normalized_lists):
+        # all normalized list names are unique, use case-insensitive comparison
+        lists = normalized_lists
+    for n in lists:
```

Comment:
I'm not sure this `for` is correct. Won't this just return the first list from those provided, rather than all of them (this never seems to return a list).